tests/test_chromatic.py

```diff
@@ -1609,15 +1569,30 @@ def test_ChromaticOpticalPSF():
     # plot(waves, bandpass(waves)*disk_SED(waves), ls=' ', marker='o', c='r', markersize=10)
     # show()
 
+    star = galsim.Gaussian(fwhm=1.e-8) * disk_SED
     psf = galsim.ChromaticOpticalPSF(lam=lam, diam=diam, aberrations=aberrations,
                                      obscuration=obscuration, nstruts=nstruts)
     do_pickle(psf)
 
+    if not os.path.isfile(os.path.join(refdir, 'r_exact.fits')):
```

Comment:
Maybe put a warning here that this is happening.  nosetests will print warnings amidst the dots, so if this happened in a nosetests run for some reason (which shouldn't really happen), it would be noticed.  Something like
```
import warnings
warnings.warn("Could not find file r_exact.fits, so generating it from scratch.  This should only "
              "happen if you intentionally deleted the file in order to regenerate it!")
```